reco_utils/common/gpu_utils.py

```diff
@@ -15,14 +19,76 @@ def get_number_gpus():
     except CudaSupportError:
         return 0
 
-    
+
 def clear_memory_all_gpus():
-    """Clear memory of all GPUs.
-    """
+    """Clear memory of all GPUs."""
     try:
         for gpu in cuda.gpus:
             with gpu:
                 cuda.current_context().deallocations.clear()
     except CudaSupportError:
         print("No CUDA available")
-        
+
+
+def get_cuda_version():
+    """Get CUDA version
+    
+    Returns:
+        str: Version of the library.
+    """
+    if sys.platform == "win32":
+        raise NotImplementedError("Implement this!")
+    elif sys.platform == "linux" or sys.platform == "darwin":
+        path = "/usr/local/cuda/version.txt"
```

Comment:
Do we want expose this to be a function parameter in case it is not installed in the default place?

Reply:
yeah that makes sense